ui/components/app/assets/defi-list/defi-list.tsx

```diff
@@ -29,6 +29,8 @@ type DefiListProps = {
   onClick: (chainId: string, protocolId: string) => void;
 };
 
+// TODO: Fix in https://github.com/MetaMask/metamask-extension/issues/31860
+// eslint-disable-next-line @typescript-eslint/naming-convention
 export default function DefiList({ onClick }: DefiListProps) {
```

Comment:
Can we doublecheck this rule for components? Components should be PascalCase?

Reply:
Discussed offline, we assumed JSX components should use arrow function syntax to avoid this
instead of `function MyComponent() {}` we use `const MyComponent = () => {}`